can't install

Can't even install via `npm install`
```
npm ERR! path E:\DEVELOPMENT UTILS\PERSONAL\GENERATIVE\node_modules\p5.capture
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c playwright install
npm ERR! 'playwright' is not recognized as an internal or external command,
npm ERR! operable program or batch file.

```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -8,6 +8,7 @@ Make sure you have Node.js and Yarn installed.
 
 1. Fork this repository and clone your fork
 2. Install dependencies: `yarn install`
+3. Install browsers for E2E test: `yarn setup`
 
 ## Build
 

--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
   "scripts": {
     "build": "tsc && vite build",
     "format": "prettier --write .",
-    "postinstall": "playwright install",
+    "setup": "playwright install",
     "prepublishOnly": "yarn build && yarn test:unit run && yarn test:e2e:skipbuild",
     "test:e2e": "playwright test",
     "test:e2e:skipbuild": "SKIP_BUILD=true yarn test:e2e",

